app/src/main/java/org/oppia/app/player/state/ImageRegionSelectionInteractionView.kt

```diff
@@ -88,7 +87,7 @@ class ImageRegionSelectionInteractionView @JvmOverloads constructor(
 
   fun setClickableAreas(clickableAreas: List<ImageWithRegions.LabeledRegion>) {
     this.clickableAreas = clickableAreas
-    // Resenting the backgrounds in case of wrong answer
+    // Resets the backgrounds for all regions if any have been loaded. This ensures the backgrounds are reset in the case when an incorrect answer is submitted.
```

Comment:
As I mentioned earlier, this needs to be line-wrapped. It exceeds the maximum characters per line (100).

Reply:
do we need to wrap comments as well? although i did that

Reply:
do we need to wrap comments as well ?

Reply:
done

Reply:
Yep, all comments need to be wrapped. Thanks for the changes.